src/courtroom.cpp

```diff
@@ -1530,6 +1518,8 @@ void Courtroom::chat_tick()
 
     ++tick_pos;
   }
+
+  ui_vp_message->repaint();
```

Comment:
According to https://doc.qt.io/qt-5/qwidget.html#repaint this forces QT to immediately redraw the widget, rather than batch together a bunch of paint operations together before actually redrawing. The current system has the issue were, if the chat tick interval is very low (e.g. 25), QT in lower-tier PCs tends to do this batching of paint operations, which results in jagged text rendering (characters appear to be rendered 1 or 2 or 3 characters at a time). Refer to video for example: 

https://user-images.githubusercontent.com/42015761/115149932-3dc50e00-a034-11eb-8474-3dde1d6b47f7.mp4

Because of the forced lower limit of chat tick interval (it cannot be under 20), we don't have to deal with the warnings about text flickering. 

The person who recorded this video reported massive improvements after the ->repaint() was added, characters being smoothly added now.